### ISSUE TITLE
Can I change background of pdf

### ISSUE THREAD

**Issue by ponetagon** [IzakMarais/reporter#160]
I want a black background. Can I do it?

**Comment by IzakMarais** [comment]
You can customise the report LaTeX template as described in the * template* section under [Query parameters](https://github.com/IzakMarais/reporter#query-parameters) in the readme. 

As to how to achieve a black report background, I am not sure how to do that in LaTeX. You would have to figure out that part on your own. Futhermore, it is not possible to request graph images with black backgrounds currently (unless you modify the reporter source code). 

In short, you could get this working but it would require some significant effort from your part.

**Comment by Zhenye-Na** [comment]
Hope you @ponetagon already find the solution, but as a follow up solution.

If you read the log carefully you will notice the url which the project use to download image

```
http://localhost:3000/render/dashboard-solo/db/your-dashboard-name?from=now-7d&height=500&panelId=4&theme=light&to=now&width=1000
```

there is a `theme` parameter, change it to `dark` and you will see the image is in dark background.

**Comment by IzakMarais** [comment]
Thanks for the tip; I had written that code so long ago I forgot about the theme.
